.bsp/sbt.json

```diff
@@ -0,0 +1 @@
+{"name":"sbt","version":"1.9.7","bspVersion":"2.1.0-M1","languages":["scala"],"argv":["/Library/Java/JavaVirtualMachines/jdk-11.jdk/Contents/Home/bin/java","-Xms100m","-Xmx100m","-classpath","/Users/roshan.gautam/Library/Application Support/JetBrains/IntelliJIdea2024.1/plugins/Scala/launcher/sbt-launch.jar","-Dsbt.script=/opt/homebrew/bin/sbt","xsbt.boot.Boot","-bsp"]}
```

Comment:
Doesn't look like something you should check in, please investigate

Reply:
Just a quick one, Is it ok just to include in the .bsp/ in gitignore and commit and push  it once again.

Reply:
Delete this file

Reply:
Thanks, I have deleted the file.